js/main.js

```diff
@@ -45,6 +48,24 @@
     });
   };
   
+  radioAllBtn.addEventListener('click', () => {
+      displayTodos(todos);
+    });
+    
+  radioWorkBtn.addEventListener('click', () => {
+      const workTodos = todos.filter(todo => {
+        todo.status === '作業中'
+      })
```

Comment:
`filter`の構文がちがっていますね。下記のように修正してみましょう。
```suggestion
      const workTodos = todos.filter(todo => 
        todo.status === '作業中'
      )
```